library/src/main/res/values/strings.xml

```diff
@@ -25,9 +25,13 @@
     <string name="stream_input_type_auto_mention">Searching for people</string>
     <string name="stream_input_camera_title">Capture Image or Video</string>
 
-    <string name="stream_camera_permission_message">needs camera and storage access so that you can take photos and videos. Please enable it in Settings.</string>
-    <string name="stream_storage_permission_message">needs storage access so that you can send and save photos, videos, music and the other files.\nPlease enable it in Settings.</string>
-    <string name="stream_both_permissions_message">needs camera and storage access so that you can take, send and save photos, videos music and the other files. Please enable it in Settings.</string>
+    <string name="stream_camera_permission_message">Camera and storage access permissions are needed in order to be able to take photos and videos</string>
+    <string name="stream_storage_permission_message">Storage access permissions is needed in order to be able to send and save photos, videos, music and the other files</string>
+    <string name="stream_camera_permission_title">Camera and Storage Access Permission</string>
+    <string name="stream_storage_permission_title">Storage Access Permissions</string>
+
+    <string name="stream_setting_message">Enable permissions on App Settings</string>
+    <string name="stream_setting_label">Settings</string>
```

Comment:
Let's be more specific with naming: `stream_setting_label` is easy to reuse somewhere accidentally and rename value eventually. Also for customers it would be hard to find specific string for translating.

`stream_input_` - all input view related strings
`stream_permissions_` - all permissions related strings
etc.

Also think that we need to replace `stream` with `chat` later. There might be a case in the future where we have common Stream library and there `stream` prefix will make sense.